Dockerfile

```diff
@@ -8,7 +8,8 @@ MAINTAINER opentrials
 
 USER root
 RUN apt-get update -yqq && \
-    apt-get install -yqq python-pip
+    apt-get install -yqq python-pip && \
+    apt-get install -yqq postgresql-client
```

Comment:
The base Docker image already adds the `jessie-backports` repository, so we should get 9.6.